src/fmu/dataio/providers/objectdata/_xtgeo.py

```diff
@@ -354,5 +359,6 @@ def get_objectdata(self) -> DerivedObjectDescriptor:
             ),
             spec=self.get_spec(),
             bbox=self.get_bbox() or None,
+            geometry=get_geometry_ref(self.dataio.geometry, self.obj) or None,
```

Comment:
Should geometry be mandatory when exporting a grid property? E.g. raise if geometry is not given?

Reply:
That will break current behaviour